students/AntonBakaiev/hw-8/task-1.js

```diff
@@ -0,0 +1,14 @@
+function searchSubstringIndex(initialString,searchingSubstring) {
+    initialString = initialString.toLowerCase();
+    searchingSubstring = searchingSubstring.toLowerCase();
```

Comment:
Please never ever ❗ ❗ ❗ declare variables without keywords. This is a bad practice and would cause an error in strict mode.